dev/com.ibm.websphere.appserver.appSecurity-3.0/bnd.bnd

```diff
@@ -13,6 +13,9 @@
 -nobundles=true
 
 -dependson: \
+	com.ibm.websphere.appserver.cdi-2.0;version=latest, \
+	com.ibm.websphere.appserver.jaspic-1.1;version=latest, \
+	com.ibm.websphere.appserver.servlet-4.0;version=latest, \
```

Comment:
I think that Glenn recently went through and removed "feature" dependencies like these and replaced them with just bundle dependencies?

Reply:
yep you're right.  I checked with @gmarcy and he confirmed that these were removed on purpose because they caused performance issues with workspace building.  Will remove these